src/components/AccountModule/SyncedInfo.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react'
+import { GU } from '@aragon/ui'
+import { getWeb3Instances } from './utils'
+
+const SyncedInfo = () => {
+  const [lastBlockIndex, setLastBlockIndex] = useState()
+
+  useEffect(() => {
+    const { walletWeb3 } = getWeb3Instances()
+
+    async function getLastBlockNumber() {
+      const latest = await walletWeb3.eth.getBlockNumber()
```

Comment:
Also not sure about this; we should continue polling for this (or ask for a subscription to new block headers) from a higher level.

Reply:
Done in https://github.com/aragon/aragon/pull/1194 (I forgot about this :speak_no_evil:)